doc/book/part1/part1_getting_off_the_ground.rst

```diff
@@ -694,7 +694,7 @@ Fibonacci
 
 Here's a doubly recursive function::

```

Comment:
I think the `::` should be changed to a single `:` to prevent the the snippet below from being interpreted as a code block. Otherwise I get this:
`/home/guido/r/fstar/master/doc/book/part1/part1_getting_off_the_ground.rst:698: ERROR: Unexpected indentation.`

<img width="364" alt="image" src="https://github.com/FStarLang/FStar/assets/4195583/e3478197-a542-4fcd-a52c-d655b3da588a">

Reply:
Ah, gotcha, thanks - fixed!